pkg/testcase/node.go

```diff
@@ -34,7 +36,7 @@ func TestNodeStatus(
 		}
 
 		return true
-	}, "2500s", "10s").Should(BeTrue(), func() string {
+	}, "3600s", "10s").Should(BeTrue(), func() string {
```

Comment:
On a side note. I feel this Eventually loop is very cumbersome/non-useful in that it simply sits for a really long time and doesnt really tell you much as to what actually failed. At the very least the timeout value should end up being configurable... since most tests need to wait for a shorter time. i think the upgrades needed it to be longer for 2500s. and not we are upping this wait time for all tests here to 3600s. That's a long time to wait to get a failure that does not tell you much.

Reply:
has this ever taken longer than 2500s for it to come up in the case of airgap?

Reply:
I can change it back to 2500s as I added wait times in the install code... agree that we should log if anything fails but given the nature of our product (rke2 espcially), I think it is fair to wait for the state to be proper before exiting on prompt failures